libvirt/tests/src/virtual_network/iface_update.py

```
@@ -236,14 +236,30 @@ def run(test, params, env):
+            # aarch64/EFI guests often already use per-device disk boot and
+            # lack <os>/<boot>; set the os/boot precondition explicitly.
+            # (Do not rely on a cfg flag — avocado may drop unknown keys.)
+            prep_xml = vm_xml.VMXML.new_from_inactive_dumpxml(vm_name)
+            prep_xml.remove_all_boots()
+            prep_xml.setup_attrs(os={'boots': ['hd']})
+            prep_xml.sync()
+        elif disk_boot:
             update_vm_boot_order(vm_name, disk_boot)
 
         if case == 'update_driver_iommu_ast':
             iommu_attrs = eval(params.get('iommu_attrs', '{}'))
```

Ruff (0.15.21)

[error] Use of possibly insecure function; consider using `ast.literal_eval`

(S307)